lib/mobility-core/src/Kernel/Storage/Esqueleto/Transactionable.hs

```diff
@@ -104,4 +116,26 @@ runNoTransactionIO logEnv dbEnv (SqlDB run) = do
         SqlDBEnv
           { currentTime = now
           }
-  runLoggerIO logEnv $ runSqlPoolNoTransaction (runReaderT run sqlDBEnv) dbEnv.connPool Nothing
+  runLoggerIO logEnv $ do
+    res <- try @_ @SomeException $ runSqlPoolNoTransaction (runReaderT run sqlDBEnv) dbEnv.connPool Nothing
+    fromEitherOrM res $ \e -> do
```

Comment:
can be simplified:

`    catch @_ @SomeException (runSqlPool (runReaderT run sqlDBEnv) dbEnv.connPool) $ \e -> do`
then `fromEitherOrM` not required